runtime/src/lib.rs

```diff
@@ -861,12 +865,26 @@ construct_runtime!(
         UncheckedExtrinsic = UncheckedExtrinsic
```

Comment:
`NodeBlock` is different here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L451, maybe because more recent version

Reply:
Good Catch. I have addressed this.